manifests/charts/istiod-remote/templates/endpoints.yaml

```diff
@@ -3,7 +3,7 @@
 apiVersion: v1
 kind: Endpoints
 metadata:
-  {{- if .Values.pilot.enabled }}
+  {{- if .Values.enabled }}
```

Comment:
How is this actually used? Does a user installing the remote chart actually decide between these two values, or did we just copy-paste from `base` and not change the chart to make sense?

Reply:
This part seems copied over from the base chart - istiod-remote is effectively a combination of both base and istiod. Long-term, @bleggett and I were planning some additional changes to consolidate some of the installation params and trim some of the unnecessary components. I think for istiod-remote we can always assume that pilot.enabled is false (and configCluster is always true?), but right now we still need to explicitly set it.

Reply:
How `base` and `istiod-remote` are actually supposed to be used (and how people are supposed to know how to use them properly) in a multi-cluster+revisioned context is still a bit opaque to me.